### ISSUE TITLE
Update failing on Fedora 38 even after following the guidelines

### ISSUE THREAD

**Issue by prajwel** [owncloud/client#10854]
### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Describe the bug

Updating Fedora 38 fails with the following error

``` 
$ sudo rpm --import https://download.owncloud.com/desktop/ownCloud/daily/4.0/linux/Fedora_38/repodata/repomd.xml.key
$ sudo dnf upgrade --refresh -y
....
owncloud                                                                                                                                                                                                      2.4 kB/s | 1.4 kB     00:00    
GPG key at https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key (0xFD41A71A) is already installed
The GPG keys listed for the "owncloud" repository are already installed but they are not correct for this package.
Check that the correct key URLs are configured for this repository.. Failing package is: owncloud-client-4.0.0-10896.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for owncloud-client-nautilus-4.0.0-10896.x86_64.rpm is not trusted. Failing package is: owncloud-client-nautilus-4.0.0-10896.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for owncloud-client-overlays-icons-4.0.0-10896.x86_64.rpm is not trusted. Failing package is: owncloud-client-overlays-icons-4.0.0-10896.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: GPG check FAILED
```

### Expected behavior

owncloud should be updated.

### Client version number

ownCloud ownCloud 3.2.1.10355
https://github.com/owncloud/client/commit/9724d662fd9a668a981c58a7db7c7231a4c5bfe1
Libraries Qt 5.15.2, OpenSSL 3.0.8 7 Feb 2023
Using virtual files plugin: suffix
fedora-6.2.13-300.fc38.x86_64

### Desktop environment (Linux only)

Fedora 38

**Comment by einarjh** [comment]
This issue (and the closed one at https://github.com/owncloud/client/issues/10839) is probably caused by the packages being signed with the wrong key (at least not the key that is distributed at the moment). I tested on a brand new VM, no pre-installed software of any kind, and followed the steps in the installation guide, and got the following error:

```
GPG key at https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key (0xFD41A71A) is already installed
The GPG keys listed for the "owncloud" repository are already installed but they are not correct for this package.
Check that the correct key URLs are configured for this repository.. Failing package is: ocqt5152-libQt5Core5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5DBus5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5DBus5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5Gui5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5Gui5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5Network5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5Network5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5PrintSupport5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5PrintSupport5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5Sql5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5Sql5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5Widgets5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5Widgets5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libqt5-qtsvg-5.15.2-3.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libqt5-qtsvg-5.15.2-3.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libqt5-qttranslations-5.12.10-3.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libqt5-qttranslations-5.12.10-3.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libqt5-qtwayland-5.15.2-4.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libqt5-qtwayland-5.15.2-4.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-qt5keychain1-0.12.0-7.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-qt5keychain1-0.12.0-7.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for owncloud-client-4.0.0-10896.x86_64.rpm is not trusted. Failing package is: owncloud-client-4.0.0-10896.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: GPG check FAILED
```

**Comment by einarjh** [comment]
Note, that was when trying to install the package in the first place. As of now, it is not possible to install the owncloud client on a fresh Fedora 38 machine.

**Comment by michaelstingl** [comment]
Communication about progress in the community forum:
- https://central.owncloud.org/t/owncloud-client-update-problem-on-fedora-38/42230

**Comment by fmoc** [comment]
We published a key file that can be imported and fixes the problems in the thread linked on the forum. See https://central.owncloud.org/t/owncloud-client-update-problem-on-fedora-38/42230/16?u=fmoc for all the details.

A permanent solution is being worked on.

**Comment by fmoc** [comment]
We now use a new key (`F05F7DD7953A07DF36579DAA498C45EBE94E7B37`, `ownCloud Client Team (Signing Key) <info@owncloud.com>`) to sign Fedora 38+ builds. In the future (i.e., starting with release 5.0) we are going to use this key for all builds.

You can install the latest 4.1 dailies as a preview (https://download.owncloud.com/desktop/ownCloud/daily/4.1/linux/download/). 4.1 will be released shortly.

**Comment by saw-jan** [comment]
Install desktop-client (`4.1`) on fresh Fedora38 works :heavy_check_mark: 
```bash
# Fedora release 38 (Thirty Eight)
# desktop: Xfce 4.18

$ sudo rpm --import https://download.owncloud.com/desktop/ownCloud/daily/4.1/linux/Fedora_38/repodata/repomd.xml.key
$ sudo dnf config-manager --add-repo https://download.owncloud.com/desktop/ownCloud/daily/4.1/linux/Fedora_38/owncloud.repo
$ sudo dnf clean all
$ sudo dng install desktop-client
...
Fedora 38 - x86_64                                                                                                                                            629 kB/s | 1.6 kB     00:00    
Importing GPG key 0xEB10B464:
 Userid     : "Fedora (38) <fedora-38-primary@fedoraproject.org>"
 Fingerprint: 6A51 BBAB BA3D 5467 B617 1221 809A 8D7C EB10 B464
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-38-x86_64
Is this ok [y/N]: y
Key imported successfully
...
Installed:
  SDL2-2.26.3-1.fc38.x86_64                         libdecor-0.1.1-2.fc38.x86_64                libgnome-keyring-3.12.0-25.fc38.x86_64     libxkbcommon-x11-1.5.0-2.fc38.x86_64              
  ocqt5152-libQt5Core5-5.15.2-41.1.x86_64           ocqt5152-libQt5DBus5-5.15.2-41.1.x86_64     ocqt5152-libQt5Gui5-5.15.2-41.1.x86_64     ocqt5152-libQt5Network5-5.15.2-41.1.x86_64        
  ocqt5152-libQt5PrintSupport5-5.15.2-41.1.x86_64   ocqt5152-libQt5Sql5-5.15.2-41.1.x86_64      ocqt5152-libQt5Widgets5-5.15.2-41.1.x86_64 ocqt5152-libqt5-qtsvg-5.15.2-3.1.x86_64           
  ocqt5152-libqt5-qttranslations-5.12.10-3.1.x86_64 ocqt5152-libqt5-qtwayland-5.15.2-4.1.x86_64 ocqt5152-qt5keychain1-0.12.0-7.1.x86_64    owncloud-client-4.1.0-0.daily20230605.11169.x86_64
  pcre2-utf16-10.42-1.fc38.1.x86_64                 tslib-1.22-8.fc38.x86_64                    xcb-util-image-0.4.1-2.fc38.x86_64         xcb-util-keysyms-0.4.1-2.fc38.x86_64              
  xcb-util-renderutil-0.3.10-2.fc38.x86_64          xcb-util-wm-0.4.2-2.fc38.x86_64            

Complete!
```

But trying to install latest desktop-client (`4.0`) fails:
```bash
$ sudo rpm --import https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
$ sudo dnf config-manager --add-repo https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/owncloud.repo
$ sudo dnf clean all
$ sudo dng install desktop-client
...
Importing GPG key 0xEB10B464:
 Userid     : "Fedora (38) <fedora-38-primary@fedoraproject.org>"
 Fingerprint: 6A51 BBAB BA3D 5467 B617 1221 809A 8D7C EB10 B464
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-38-x86_64
Is this ok [y/N]: y
Key imported successfully
owncloud                                                                                                                                                      2.3 kB/s | 1.4 kB     00:00    
GPG key at https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key (0xFD41A71A) is already installed
The GPG keys listed for the "owncloud" repository are already installed but they are not correct for this package.
Check that the correct key URLs are configured for this repository.. Failing package is: ocqt5152-libQt5Core5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5DBus5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5DBus5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5Gui5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5Gui5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5Network5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5Network5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5PrintSupport5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5PrintSupport5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5Sql5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5Sql5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libQt5Widgets5-5.15.2-41.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libQt5Widgets5-5.15.2-41.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libqt5-qtsvg-5.15.2-3.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libqt5-qtsvg-5.15.2-3.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libqt5-qttranslations-5.12.10-3.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libqt5-qttranslations-5.12.10-3.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-libqt5-qtwayland-5.15.2-4.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-libqt5-qtwayland-5.15.2-4.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for ocqt5152-qt5keychain1-0.12.0-7.1.x86_64.rpm is not trusted. Failing package is: ocqt5152-qt5keychain1-0.12.0-7.1.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
Public key for owncloud-client-4.0.0-10896.x86_64.rpm is not trusted. Failing package is: owncloud-client-4.0.0-10896.x86_64
 GPG Keys are configured as: https://download.owncloud.com/desktop/ownCloud/stable/latest/linux/Fedora_38/repodata/repomd.xml.key
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: GPG check FAILED
```

**Comment by fmoc** [comment]
Which is the expected result. Thanks for testing @saw-jan!

**Comment by JoKalliauer** [comment]
Confirm fix with 4.1:

```sh
sudo rm /etc/yum.repos.d/owncloud.repo
sudo rpm --import https://download.owncloud.com/desktop/ownCloud/daily/4.1/linux/Fedora_38/repodata/repomd.xml.key
sudo dnf config-manager --add-repo https://download.owncloud.com/desktop/ownCloud/daily/4.1/linux/Fedora_38/owncloud.repo
sudo dnf clean all
sudo dnf install owncloud-client
sudo dnf upgrade owncloud-client
```
on:
```sh
     Static hostname: lws84.imws.tuwien.ac.at
           Icon name: computer-laptop
             Chassis: laptop 💻
          Machine ID: 96434f58f7644401a3f9d69da903899d
             Boot ID: 513a16b876bc4b11bf5f345a9d3fbe93
    Operating System: Fedora Linux 38 (Workstation Edition)
         CPE OS Name: cpe:/o:fedoraproject:fedora:38
      OS Support End: Tue 2024-05-14
OS Support Remaining: 11month
              Kernel: Linux 6.3.6-200.fc38.x86_64
        Architecture: x86-64
     Hardware Vendor: Lenovo
      Hardware Model: ThinkPad P15 Gen 1
    Firmware Version: N30ET49W (1.32 )
       Firmware Date: Wed 2022-12-14
```

@saw-jan  please change the typo `sudo dng install desktop-client` to `sudo dnf install owncloud-client` thanks

**Comment by fmoc** [comment]
Fixed the remaining occasions.
